bycycle/features/burst.py

```diff
@@ -54,6 +54,18 @@ def compute_burst_features(df_shape_features, df_samples, sig,
         When dual threshold burst detection is used (i.e. burst_method == 'amp'):
 
         - ``burst_fraction`` : fraction of a cycle that is bursting
+
+    Examples
+    --------
+    Determine burst features.
```

Comment:
This might just be me, but 'determine' feels like a slightly odd word here. 
Maybe 'Compute'?